docs/howtos/Client Generation/11etag.mdx

```diff
@@ -0,0 +1,64 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Conditional Request Header
+
+This doc details what emitters will generate for conditional request header
+
+## `If-Match` and `If-None-Match`
```

Comment:
How about `If-Modified-Since` and `If-Unmodified-Since`?

I think we have cases of either 2 or 4 headers.

Reply:
Will add for the other 2 headers later.